integration/upload_download.test.ts

```diff
@@ -171,6 +171,19 @@ describe(`Upload and download end-to-end tests for portal '${portal}'`, () => {
     expect(contentType).toEqual("application/octet-stream");
   });
 
+  it("getFileContentBinary should get binary data", async () => {
+    const file = new File(["foo"], "bar");
+    const { skylink } = await client.uploadFile(file);
+
+    // Get file content and check returned values.
+
+    const { data, contentType } = await client.getFileContentBinary(skylink);
+
+    expect(typeof data).toEqual("object");
+    expect(data instanceof ArrayBuffer).toBeTruthy();
+    expect(contentType).toEqual("application/octet-stream");
```

Comment:
1. How does this differ from the one above? I'm especially confused by the response type in the old test also being `arraybuffer`. If I understand the code correctly, this would make `getFileContent` (as called on line 167) to behave in the same way as `getFileContentBinary`.

2. Shouldn't we verify the content of the file as well? If we had that before, we wouldn't have been surprised that Axios corrupts binary data.

Reply:
Good observations, updated.